README.md

```diff
@@ -1,45 +1,53 @@
 
 # CS4295_Project
-Final project for CS4295 Release Engineering for Machine Learning Applications
+Final project for CS4295 Release Engineering for Machine Learning Applications.
 
 
-1. download_dataset
-2. preprocessing
-3. model_train
-4. model_predict
-
 ## Setup
 To set up the project, follow these steps:
 
 1. Clone the repository:
     ```bash
-    git clone https://github.com/nickdubbel/CS4295_Project.git
+    git clone https://github.com/Release-Engineering-Group-13/CS4295_FinalProject.git
     ```
 
 2. Create and activate a virtual environment:
     ```bash
```

Comment:
Instead it is also possible to let Poetry create a virtual environment, it think that might be easier
```suggestion
2. Install Poetry
    ```bash
```